cypress/e2e/api/customers/getCustomers.cy.js

```diff
@@ -0,0 +1,140 @@
+const API_URL = Cypress.env('API_URL');
+
+describe('Get Customers', () => {
+  it('returns the correct status and body structure', () => {
+    cy.request({
+      method: 'GET',
+      url: `${API_URL}/customers`,
+    }).then(({ status, body }) => {
+      expect(status).to.eq(200);
+      const customers = body.customers;
+      customers.forEach((customer) => {
+        expect(customer).to.have.property('id');
```

Comment:
Outra alternativa seria algo assim:

`expect(customer).to.have.all.keys('id', 'name', 'employees', 'size', 'industry', 'contactInfo', 'address')`.

Reply:
Mas gosto da sua solução.

Ou seja, não precisa mudar.

Só pra demonstrar uma alternativa mesmo.